### PR TITLE
[10.0][ADD] partner_contact_type_end_user

### PR DESCRIPTION
**Pull request by rousseldenis**
image

### CODE REVIEW

**Comment by rousseldenis** [comment]
This is related to https://github.com/OCA/sale-workflow/pull/763

**Review by CarlosRoca13 — APPROVED** [review]
Code LGTM 
Runbot don't works, but I see your picture :+1:

File: partner_contact_type_end_user/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (NL66278) [inline partner_contact_type_end_user/__manifest__.py:3]: Make license lines https:

File: partner_contact_type_end_user/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+{
+    'name': 'Partner Contact Type End User',
```
Comment (NL66278) [inline partner_contact_type_end_user/__manifest__.py:6]: As this is a completely new module, please apply black conventions on them, easing later migration to 13.0.

File: partner_contact_type_end_user/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    'summary': """
```
Comment (NL66278) [inline partner_contact_type_end_user/__manifest__.py:7]: Summary should be one-liner, so do not use triple quotes.

**Review by NL66278 — CHANGES_REQUESTED** [review]
:+1:  In general looks good to me. But a few changes would be nice. See other comments.

**Comment by NL66278** [comment]
@rousseldenis Would you please take up the comments? I would like to merge this.

**Comment by NL66278** [comment]
Closing as questions not answered.

**Pull request closed without merging**